### PR TITLE
Create link_adobe_share_unsoliticed_sender.yml

### PR DESCRIPTION
**Pull request by zoomequipd**
# Description

Add new coverage from observed abused Adobe message which link to an Adobe hosted file which links to cred phishing. 

# Associated samples

Sample was provided privately

### CODE REVIEW

**Comment by zoomequipd** [comment]
looking good, very low volume based on telemetry data

File: detection-rules/link_adobe_share_unsoliticed_sender.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: "Link: Adobe Share from Unsolicited Sender"
+description: "This attack surface reduction rule matches on message from Adobe which was sent by an email address (as determined by the sender display name) which doesn't appear to have a relationship with the recipient organization."
```
Comment (aidenmitchell) [inline detection-rules/link_adobe_share_unsoliticed_sender.yml:2]:
```suggestion
description: "This attack surface reduction rule matches on messages from Adobe which were sent by an email address (as determined by the sender display name) which doesn't appear to have a relationship with the recipient organization."
```

**Review by aidenmitchell — CHANGES_REQUESTED** [review]

**Comment by zoomequipd** [comment]
going to rework this based on customer feedback

**Review by aidenmitchell — APPROVED** [review]

**Pull request merged**